implementations/rust/ockam/ockam_command/src/policy/list.rs

```diff
@@ -40,21 +44,25 @@ impl ListCommand {
         let node = BackgroundNodeClient::create(ctx, &opts.state, &self.at).await?;
         let is_finished: Mutex<bool> = Mutex::new(false);
 
-        let resource = &self.resource;
+        let resource_type = if self.resource_type.is_none() && self.resource.is_none() {
+            Some(&ResourceType::TcpOutlet)
```

Comment:
Could you please elaborate why is that true?

Reply:
All the list commands have a default target. I've added this fallback value to keep the UX consistent and not return an error when the user runs `ockam policy list`.

Reply:
Shouldn't we list all policies (for all resources and resource types) in that case? I think we discussed that

Reply:
Shouldn't we list all policies (for all resources and resource types) in that case? I think we discussed that

Reply:
True, I'll include that.